### PR TITLE
Added setCornerRadius function to avoid `Ambiguous use of 'cornerRadius'` issue

### PR DESCRIPTION
**Pull request by flowdeskadmin**

### CODE REVIEW

File: Source/WSTagsField.swift [hunk 1]
```diff
@@ -331,6 +331,11 @@ open class WSTagsField: UIScrollView {
         repositionViews()
     }
 
+    /// Set corner radisu of tag views
```
Comment (ricardopereira) [inline Source/WSTagsField.swift:334]: Small typo in the comment.

**Review by ricardopereira — CHANGES_REQUESTED** [review]

File: Source/WSTagsField.swift (hunk 1 above)
Comment (mobilestar1) [reply Source/WSTagsField.swift:334]: Sorry for typo. Thanks.

**Comment by mobilestar1** [comment]
@ricardopereira Please check again.

**Comment by ricardopereira** [comment]
I'll accept this for now but maybe it should have a more dedicated method for tag customization. Something like `appearance` for `TagView`'s.

**Comment by ricardopereira** [comment]
@mobilestar1 Actually, there's already a method for the same purpose 😅
https://github.com/whitesmith/WSTagsField/blob/6553752451d13642c42f470486e89776ddccbf83/Source/WSTagsField.swift#L90-L94

**Comment by mobilestar1** [comment]
> @mobilestar1 Actually, there's already a method for the same purpose 😅
> https://github.com/whitesmith/WSTagsField/blob/6553752451d13642c42f470486e89776ddccbf83/Source/WSTagsField.swift#L90-L94
> 
> Sorry, I should've checked this earlier.

Of course, I knew that. As You can see from PR title I added `setCornerRadius` function to avoid `Ambiguous use of 'cornerRadius'` issue.
Several libraries declared cornerRadius variable publicly so it conflicts each other.

**Comment by ricardopereira** [comment]
@mobilestar1 Ok, could you please share an example project for further investigation? This situation can happen with any property. For example with the `text` property: if someone adds an extension with `UIView.text` then it should not clash with `wsTextField.text`, right? We don't need to create a `setText` just to avoid that particular clash. That would be impractical.

**Comment by mobilestar1** [comment]
For example, in my project I added this UIView extension to set corner radius easily.(I checked many other libraries and they also provide this solution)

```
extension UIView {

    @IBInspectable
    var cornerRadius: CGFloat {
        get {
            return layer.cornerRadius
        }
        set {
            layer.cornerRadius = newValue
        }
    }
}
```

Now if I want to set tag field corner radius by tagField.cornerRadius=x, then I got `Ambigious use of cornerRadius` issue.

**Comment by mobilestar1** [comment]
any updates?

**Comment by ricardopereira** [comment]
I think it would make more sense to rename the `cornerRadius` property to `tagsCornerRadius`. It would solve the issue but doing that the `borderWidth`, `borderColor`, etc. need to be renamed too (`tagsBorderWidth`, `tagsBorderColor`...), to keeps things consistent. But the best would be some sort of tags configuration mechanism.

Even so, I'll merge this but keep in mind that it will break in the future. Thanks

**Comment by aashishSapkotaKC** [comment]
> I think it would make more sense to rename the `cornerRadius` property to `tagsCornerRadius`. It would solve the issue but doing that the `borderWidth`, `borderColor`, etc. need to be renamed too (`tagsBorderWidth`, `tagsBorderColor`...), to keeps things consistent. But the best would be some sort of tags configuration mechanism.
> 
> Even so, I'll merge this but keep in mind that it will break in the future. Thanks

Yes Please do change the borderColor property name likewise, I'm facing the ambiguous issue in borderColor.

**Pull request merged**